src/google/protobuf/descriptor.cc

```diff
@@ -2612,10 +2612,12 @@ void FileDescriptor::CopyHeadingTo(FileDescriptorProto* proto) const {
   }
 
```

Comment:
You're missing https://github.com/protocolbuffers/protobuf/commit/9b218c1d9aaa8108f65cca0d65325936a8bbeac6 I think

Reply:
Pulled it in, thanks for the catch!